docker/bin/logs

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```

Comment:
Here and below, I'd add `set -o errexit` at the top of the script: Shell scripts don't exit-on-error by default. 

There are [more options](https://sipb.mit.edu/doc/safe-shell/), but I think this is the most important.

Reply:
Thank you very much for this! I have added this and specified you as a co-author.